src/System.Management.Automation/engine/Subsystem/FeedbackSubsystem/FeedbackHub.cs

```diff
@@ -11,13 +11,14 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.PowerShell.Commands;
+using static System.Net.Mime.MediaTypeNames;
```

Comment:
```suggestion
```

I think this was a code action trying to be helpful unless I'm missing where it gets used

Reply:
Good catch. Thanks!